### ISSUE TITLE
WoW Classic Version?

### ISSUE THREAD

**Issue by miketheitguy** [miketheitguy/Vendomatic#4]
I deleted almost all of the old revisions, so I would have to re-write some code for WoW Classic :)

**Comment by barryhalls** [comment]
I also would love a version for classic please!

**Comment by miketheitguy** [comment]
Agreed. I switched jobs recently and have spent a lot of work on that. Especially around software development!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by miketheitguy** [comment]
If you enable it for classic it should just work. If it doesn’t, add an issue here!!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by miketheitguy** [comment]
I’ve got a few things I’m working on related to work this weekend. So I might put this on the list for next weekend.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by barryhalls** [comment]
It doesn't let me drag my reagents to auto buy in because there is no box... can we use a text string or manually edit our config? If so how? I tried manually everything my rune of teleportation but must not know all the settings to get it to work properly.

**Comment by miketheitguy** [comment]
Crap. I actually think I fixed that and this might be a regression. I’ll take a look tomorrow.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by barryhalls** [comment]
Thanks Mike for taking your time to look into this

**Comment by JohanLeirnes** [comment]
@miketheitguy did you manage to fix the bug with the box not showing?

**Comment by barryhalls** [comment]
I switched to stealyourcarbon, it works for now.
